src/app/story/storyrow/storyrow.component.css

```diff
@@ -50,11 +42,77 @@
     margin: 0;
 }
 
+.node-display-name {
+    font-weight: bold;
+    width: 134px;
+}
+
+.node-timestamp {
+    font-size: 1.1rem;
+    width: 134px;
+}
+
+.clock-icon {
+    /* We want it to be on top of everything except the sidebar */
+    z-index: 97;
+    margin-left: 40px;
```

Comment:
We use rem everywhere else if I remember correctly. Could you use rem units in your code too please?